jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -209,11 +211,15 @@ def __call__(self, add_data=True):
     def slice_display_unit_name(self):
         return 'spectral'
 
+    @property
+    def slice_indicator_viewers(self):
+        return [v for v in self.app._viewer_store.values() if isinstance(v, WithSliceIndicator)]
+
     @observe('active_step', 'is_active')
     def _active_step_changed(self, *args):
         self.aperture._set_mark_visiblities(self.active_step in ('', 'ap', 'ext'))
         self.background._set_mark_visiblities(self.active_step == 'bg')
-        self.marks['bg_spec'].visible = self.active_step == 'bg'
+        self.marks['bg_ext'].visible = self.active_step == 'bg'
```

Comment:
Is `ext` for "extraction", rather than its usual "extension" as abbreviated elsewhere in jdaviz? This would require changes throughout the PR, but could we use a less degenerate abbreviation? Maybe `extr`?

Reply:
(While I'm aware that I can be a pedant,) more descriptive names are always better.